src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/contactperson/ContactPersonResource.kt

```diff
@@ -219,3 +221,41 @@ data class Identifier(
   @Schema(description = "The issued authority", example = "Police")
   val issuedAuthority: String?,
 )
+
+@Schema(description = "The data held in NOMIS about a person's contact with a prisoner")
+@JsonInclude(JsonInclude.Include.NON_NULL)
+data class PersonContact(
+  @Schema(description = "Unique NOMIS sequence for this identifier for this contact")
+  val id: Long,
+  @Schema(description = "The contact type")
+  val contactType: CodeDescription,
+  @Schema(description = "The relationship type")
+  val relationshipType: CodeDescription,
+  @Schema(description = "True if active")
+  val active: Boolean,
+  @Schema(description = "Date contact is no longer active")
+  val expiryDate: LocalDate?,
+  @Schema(description = "True if approved to visit the prisoner")
+  val approvedVisitor: Boolean,
+  @Schema(description = "True if next of kin to the prisoner")
+  val nextOfKin: Boolean,
+  @Schema(description = "True if emergency contact for the prisoner")
+  val emergencyContact: Boolean,
+  @Schema(description = "Free format comment text")
+  val comment: String?,
+  @Schema(description = "The prisoner this person is a contact for")
+  val prisoner: ContactForPrisoner,
+)
+
+@Schema(description = "The data held in NOMIS about a person's contact with a prisoner")
+@JsonInclude(JsonInclude.Include.NON_NULL)
+data class ContactForPrisoner(
```

Comment:
No idea what is needed yet, but this is useful for debugging endpoint if you know some basic details about the prisoner